src/test/resources/features/oclsubscriptionmodule/index.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Import Dictionary via subscription module
```

Comment:
any reason for naming the feature file as `index`?

Reply:
Just because it was the only file in the folder I named it index but I can give it another name.